runtimelib/src/jupyter/client.rs

```diff
@@ -140,13 +193,14 @@ impl JupyterClient {
         }
     }
 
-    pub async fn send(&mut self, message: JupyterMessage) -> Result<JupyterMessage, Error> {
+    /// Send a message to the kernel, and return the response
```

Comment:
This should probably say like send a `*_request` message and receive the corresponding `*_reply` message. I realize the code doesn't show it below but that's what it should read as.